.gitignore

```diff
@@ -1,6 +1,6 @@
 #### Maven
 
-target/
+/target/
```

Comment:
And why not put the leading slash in front of the mysterious files that follow? If they pop up elsewhere in the project I would rather be alerted to that fact by git.